R/adjust-logrank.R

```diff
@@ -18,7 +18,7 @@ adjust.LogRank <- function(model, data, ...){
     dplyr::mutate(
       uu_cl  = .data$trt1 * (.data$O.hat - .data$adjust1) -
                .data$trt0 * (.data$O.hat - .data$adjust0),
-      ssig_l = .data$event * .data$Y0 * .data$Y1 / .data$Y^2
+      ssig_l = .data$event * .data$Y0 * .data$Y1 * (.data$Y - .data$n.events) / .data$Y^2 / (.data$Y-1)
```

Comment:
Just having a look at this change - I understand that this is a slight change of the variance calculation. Just wondering if this would be worthwhile (for `RobinCar2` mainly) to have as a second method (expert) option? Or was the old formula not behaving well and therefore should not be used anymore?

Related question, more about the code: is this then the only place this appears and needs to be changed?

Reply:
OK maybe it is just because because of the ties handling, which was not included before?

Reply:
Hi @danielinteractive, yes this change was just for handling ties. If `n.events = 1` (no ties), then the two new terms cancel out.